lib/dealership.rb

```diff
@@ -0,0 +1,41 @@
+class Dealership
+  attr_reader :name,
+              :address,
+              :inventory
+
+    def initialize(name, address)
+      @name = name
+      @address = address
+      @inventory = []
+    end
+
+    def inventory_count
+      @inventory.count
+    end
+
+    def add_car(cars)
+      @inventory << cars
+    end
+
+    def has_inventory?
+      if inventory_count == 1 || inventory_count > 1
+        true
+      else
+        false
+      end
+    end
+
+    def cars_by_make(car_make) #due to hardcoded this wont work.
+      @inventory.find_all do |correct_make|
```

Comment:
So when you are iterating through the inventory you are accessing a car which might help with some of the confusion:

```
@inventory.find_all do |car|
....
```

so now that you know you are going through the cars you can call the `make` method on them. (I realize it won't work as expected with the hard coding)

```
@inventory.find_all do |car|
  car.make == car_make
end
```

this allows us to compare the make of the cars we are iterating through with the make `car_make` that we are passing through to this method